README.md

```diff
@@ -150,25 +150,20 @@ With the Postgres container enabled, users (both students and instructors) can c
 
 ### Spawn Specific Workspace Types
 
-> **New in Version 0.8.0**: users can spawn specific workspace types by setting the `workspace_type` key when initiating the launch request. The `workspace_type` key can either be added as a query parameter or included within the request's body (when initiating an LTI launch).
-
-To enable this feature, you must set the `JupyterHub.spawner_class` to the `IllumiDeskWorkSpaceDockerSpawner` class. Like the `IllumiDeskRoleDockerSpawner` class, the `IllumiDeskWorkSpaceDockerSpawner` class inherits from the `IllumiDeskBaseDockerSpawner` class which manages the logic to assign the image to spawn based on the `workspace_type` key in the `auth_state` dictionary.
-
 Additional workspace types include:
 
 - THEIA IDE: `theia`
 - RStudio: `rstudio`
 - VS Code: `vscode`
 
-If a value is sent which is not recognized by the system, then the default Jupyter Notebook workspace is launched for the user.
 
-When specifying a workspace type with the query parameter add the key / value to the URL encoded request. For example, with LTI 1.1 launch requests the query parameter would look like so:
+When you want to use a specific workspace type you can do it by using the [user redirect](https://jupyterhub.readthedocs.io/en/stable/reference/urls.html#user-redirect) functionality in jupyterhub combined with the query parameter `next`. For example, with LTI 1.1 the launch url would look like so:
```

Comment:
When you want to use a specific workspace type simply leverage the existing [/user-redirect/](https://jupyterhub.readthedocs.io/en/stable/reference/urls.html#user-redirect) functionality available with JupyterHub combined with the query parameter `next`. For example, with LTI 1.1 the launch url would look like so: